subgraph/bin/abis.js

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
+
+const path = require('path')
+const fs = require('fs-extra')
+const abis = require('@unlock-protocol/contracts')
+const ethers = require('ethers')
+
+const abisFolderPath = path.join(__dirname, '..', 'abis')
+
+const unlockVersions = ['v11']
+const publicLockVersions = ['v7', 'v11']
```

Comment:
Are we missing the intermediate versions here?

Reply:
yep, added them all